script.js

```diff
@@ -35,21 +35,37 @@ class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      user: null };
+		user: null,
+		toast: null};
 
   }
   signIn(email, password) {
-    // sign in APIs here
-    this.setState({
-      user: {
-        email,
-        password } });
-
-
+		axios.post('http://localhost:3000/login',{
+			username: email,
+			password: password
+		})
+		.then((response) => {
+			if(response.status === 201){this.setState({
```

Comment:
At the moment, the server returns a 201, but if the server were in future to return a 200 (or 204 etc.) this piece of code would fail silently.

Reply:
catch all response now triggers a toast notification with the server's response displayed